modules/xmpp/SDPUtil.js

```diff
@@ -443,6 +443,30 @@ var SDPUtil = {
             .split(" ")
             .map(ssrcStr => parseInt(ssrcStr));
     },
+
+    /**
+     * Sets the given codecName as the preferred codec by
+     *  moving it to the beginning of the payload types
+     *  list (modifies the given mline in place)
+     * @param {object} videoMLine the video mline object from
+     *  an sdp as parsed by transform.parse
+     * @param {string} the name of the preferred codec
+     */
+    preferVideoCodec: function(videoMLine, codecName) {
```

Comment:
I'd rather we stop writing ES5 source code right now because someone will have to translate it to ES6 eventually.

I understand the consistency argument but I think it's weakened by the consistency situation within lib-jitsi-meet.

I'm perfectly OK with you being new to JavaScript. I consider myself new because I sat down studying JavaScript in May 2016.